charter.md

```diff
@@ -1,10 +1,12 @@
-[RFC 5033](https://datatracker.ietf.org/doc/rfc5033/) describes a Best Current Practice to evaluate new congestion control algorithms as Experimental or Proposed Standard RFCs. TCP was the dominant consumer of this work, and proposals were typically discussed in research groups, for example the Internet Congestion Control Research Group (ICCRG).
+Congestion control is the process a network sender uses to determine the rate at which to send data. A congestion control algorithm balances sending sufficient data to make meaningful progress for a user with avoiding filling buffers in the network and overloading the network path. Better congestion control algorithms can help improve application performance, such as latency under load, while safeguarding overall network performance for all senders.
 
-Since [RFC 5033](https://datatracker.ietf.org/doc/rfc5033/) was published, many conditions have changed. Congestion control algorithm proponents now often have the opportunity to test and deploy at scale without IETF review. The set of protocols using these algorithms has spread beyond TCP and SCTP to include DCCP, QUIC, and beyond. There is more interest in specialized use cases such as data centers and real-time protocols. Finally, the community has gained much more experience with indications of congestion beyond packet loss.
+Standardizing a congestion control algorithm can help implementers, operators, and other interested parties develop a shared understanding of how the algorithm works, how it is expected to behave in various scenarios and configurations, and can make it easier for them to suggest improvements and identify limitations.
```

Comment:
```suggestion
Specifying a congestion control algorithm can help implementers, operators, and other interested parties develop a shared understanding of how the algorithm works, how it is expected to behave in various scenarios and configurations, and can make it easier for them to suggest improvements and identify limitations.
```
Nit, but not all specifications are standards, and I catch myself most times I conflate them... :D

Reply:
Oh great catch, thank you!